implementations/rust/ockam/ockam_command/src/docs.rs

```diff
@@ -26,15 +27,15 @@ discord channel https://discord.ockam.io
 
 static SYNTAX_SET: Lazy<SyntaxSet> = Lazy::new(SyntaxSet::load_defaults_newlines);
 static HEADER_RE: Lazy<Regex> = Lazy::new(|| Regex::new("^[A-Za-z][A-Za-z0-9 ]+:$".into()));
-static THEME: Lazy<Option<Theme>> = Lazy::new(|| {
+static THEME: Lazy<Theme> = Lazy::new(|| {
     let theme_name = match TerminalBackground::detect_background_color() {
         TerminalBackground::Light => "base16-ocean.light",
         TerminalBackground::Dark => "base16-ocean.dark",
-        TerminalBackground::Unknown => return None,
+        TerminalBackground::Unknown => "base16-ocean.dark",
```

Comment:
I read your comment [here](https://github.com/build-trust/ockam/issues/7472#issuecomment-1915499284) about this change. If a user has a terminal with a white background, this would break the experience for that user. I think this was the main reason to have this `None` default.

Reply:
Maybe we could try to use this: https://github.com/dalance/termbg
Do you want to give it a try?

Reply:
HI @adrianbenavides, apologies for my late response. I integrated your additional comments :). I checkout out your suggested libraries and ran some tests, but unfortunately the color detection is unsuported on macOS it seems: 

![image](https://github.com/build-trust/ockam/assets/17552843/3989b8ff-c193-405a-80bd-1cf05fd711b0)

I will take a look at a few other libraries and hopefully we will get something useful that will detect the correct background color for macOS.

Reply:
I think that's expected because you were using one of the terminals that are not supported (VSCode). But it should be fine if we can support the main terminals.

Reply:
Apologies, I looked over that part, will run some tests again and add this library then

Reply:
Added the termbg library, tested it and it works in the macOS terminal